src/include/platform/CHIPDeviceConfig.h

```diff
@@ -88,7 +88,7 @@
  * The maximum number of events that can be held in the chip Platform event queue.
  */
 #ifndef CHIP_DEVICE_CONFIG_MAX_EVENT_QUEUE_SIZE
-#define CHIP_DEVICE_CONFIG_MAX_EVENT_QUEUE_SIZE 100
+#define CHIP_DEVICE_CONFIG_MAX_EVENT_QUEUE_SIZE 25
```

Comment:
I am not really sure if we should change this value generically (for all the platforms). Else, LGTM

Reply:
Agree that this should not be changed by *platform*. This setting actually should be *by product*.